src/components/views/messages/IBodyProps.ts

```diff
@@ -55,4 +55,7 @@ export interface IBodyProps {
     getRelationsForEvent?: GetRelationsForEvent;
 
     ref?: React.RefObject<any> | LegacyRef<any>;
+
+    // Used by EventTilePreview to disable interactions and tab indexes
```

Comment:
it would be better to define this in terms of what it *does* than what it is used for.

```suggestion
    // Set to `true` to disable interactions and tab indexes. This may be useful when displaying a preview of the event.
```

Can you be more specific about what exactly is disabled? what sort of interactions? What are tab indexes? One approach would be to give an example: "For example, video events will be shown without controls for the user to control video playback".